src/test/java/edu/umich/ctools/esb/utils/WAPITest.java

```diff
@@ -8,17 +8,17 @@
 
 public class WAPITest extends TestCase {
 
-	public void testWAPI() {
-		fail("Not yet implemented");
-	}
+//	public void testWAPI() {
```

Comment:
Why comment these out?  It is in a testcase in a test directory.  It is fine to have this ready to roll.  It won't get deployed.